code/game/objects/items/devices/radio/encryptionkey.dm

```diff
@@ -306,7 +306,7 @@
 /obj/item/device/encryptionkey/cmb
 	name = "\improper Colonial Marshal Bureau Radio Encryption Key"
 	icon_state = "cmb_key"
-	channels = list(RADIO_CHANNEL_COMMAND = TRUE, RADIO_CHANNEL_MEDSCI = TRUE, RADIO_CHANNEL_INTEL = TRUE, RADIO_CHANNEL_ALMAYER = TRUE, RADIO_CHANNEL_COLONY = TRUE)
```

Comment:
why was this removed

Reply:
Hu-

I'm guessing my branch was out of date. Give me a moment